ahiv/kafka/protocol/packet/metadata.h

```diff
@@ -77,20 +77,20 @@ struct PartitionInformation {
     leaderId = buffer.Read<int32_t>();
     leaderEpoch = buffer.Read<int32_t>();
 
-    int32_t amountOfReplicas = buffer.Read<int32_t>();
+    auto amountOfReplicas = buffer.Read<int32_t>();
     replicas.reserve(amountOfReplicas);
     for (std::size_t currentReplica = 0; currentReplica < amountOfReplicas;
          currentReplica++) {
       replicas.emplace_back(buffer.Read<int32_t>());
     }
 
-    int32_t amountOfIsr = buffer.Read<int32_t>();
+    auto amountOfIsr = buffer.Read<int32_t>();
```

Comment:
Please use the full form of ISR. And maybe append `amount` instead of prepending `amountOf`

Reply:
I don't really know what ISR means, the kafka documentation only refers to it as "isr_nodes" and "ISR"

Reply:
ISR: InSync Replica